src/main/java/brokeculator/command/AddExpenseFromFileCommand.java

```diff
@@ -17,7 +17,7 @@ public void execute(Dashboard dashboard) {
             Expense expense = Expense.getExpenseFromFile(this.fileString);
             dashboard.getExpenseManager().add(expense);
         } catch (Exception e) {
-            UI.print("Expense cannot be added");
+            UI.println("Expense cannot be added");
```

Comment:
I think it would be good if we can define when to use UI.print and when to use the UI.prettyPrint function

Reply:
I think feedback after a command should be prettyPrint. I think we'll end up refactoring most of the Error handling so I left that how it is for now